Source/WebCore/css/make-css-file-arrays.pl

```diff
@@ -67,11 +67,11 @@
     # Write out a C array of the characters.
     my $length = length $text;
     if ($in =~ /(\w+)\.css$/) {
-        print HEADER "extern const char ${name}UserAgentStyleSheet[${length}];\n";
-        print OUT "extern const char ${name}UserAgentStyleSheet[${length}] = {\n";
+        print HEADER "extern const std::array<const char, ${length}> ${name}UserAgentStyleSheet;\n";
```

Comment:
Is the array null terminated?

Reply:
No, please read commit log.

Reply:
Checking the bug description and crash log and I got it. So the key is `makeString` gets length with `strlen` on char* input (so it needs to be null terminated), but not for span (so array does not need to be null terminated).